sync/src/tests/util.rs

```diff
@@ -45,44 +46,38 @@ pub fn generate_blocks(
     }
 }
 
-pub fn inherit_block(shared: &Shared, parent_hash: &H256) -> BlockBuilder {
+pub fn inherit_block(shared: &Shared, parent_hash: &Byte32) -> BlockBuilder {
     let parent = shared.store().get_block(parent_hash).unwrap();
-    let parent_epoch = shared.get_block_epoch(parent_hash).unwrap();
+    let parent_epoch = shared.store().get_block_epoch(parent_hash).unwrap();
```

Comment:
`shared.get_block_epoch`

Reply:
don't use `ChainProvider::get_block_epoch`, it will be deteted later